tests/smoke/smoke_test.go

```diff
@@ -19,36 +19,44 @@ package smoke
 import (
 	"testing"
 
+	"github.com/coreos/pkg/capnslog"
+
+	"github.com/rook/rook/tests/framework/clients"
 	"github.com/rook/rook/tests/framework/enums"
 	"github.com/rook/rook/tests/framework/installer"
 	"github.com/rook/rook/tests/framework/utils"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 )
 
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "smoketest")
+
 func TestSmokeSuiteK8s(t *testing.T) {
-	suite.Run(t, new(SmokeSuite))
+	suite.Run(t, new(SmokeSuitek8s))
 }
 
-type SmokeSuite struct {
+type SmokeSuitek8s struct {
```

Comment:
why rename the type to k8s? When standalone tests are added, they could have a different type name, but I would suggest we not use the k8s suffix since that's our more important target